ush/python/pygfs/task/archive.py

```diff
@@ -374,7 +374,8 @@ def _rename_cyclone_expt(arch_dict) -> None:
         if len(arch_dict.PSLOT) > 4:
             pslot4 = arch_dict.PSLOT[0:4].upper()
 
-        track_dir = arch_dict.COM_ATMOS_TRACK
+        track_dir_in = arch_dict.COMIN_ATMOS_TRACK
+        track_dir_out = arch_dict.COMOUT_ATMOS_TRACK
```

Comment:
this change needs to be reflected on lines 383, 397. for e.g. Otherwise, those lines will fail.

Reply:
It is not clear to me what you are asking or referring to. Please clarify.

Reply:
`track_dir` was replaced with `track_dir_in` and `track_dir_out`. All the subsequent references to `track_dir` need to be changed to one of those two.

Reply:
changes were made in a later commit.